zipline/algorithm.py

```diff
@@ -749,9 +749,10 @@ def symbol(self, symbol_str):
         _lookup_date = self._symbol_lookup_date if self._symbol_lookup_date is not None \
             else self.sim_params.period_end
 
-        return self.asset_finder.lookup_symbol_resolve_multiple(
+        return self.asset_finder.lookup_symbol(
             symbol_str,
-            as_of_date=_lookup_date
+            as_of_date=_lookup_date,
+            default_None=False,
```

Comment:
Does that mean that every time we don't symbol map correctly, it's going to raise an error? tl;dr we need to try/except SymbolNotFound for our calls?